### DISCUSSION TITLE
[Enhancement] [image] image组件使用preview属性时，无法监听image打开图片的事件

### DISCUSSION THREAD

**Discussion by wydingez** [element-plus/element-plus#8919] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `image`

## Description

目前项目中有一个弹框（弹框中按钮【上一条/下一条】会监听用户的键盘方向左键和右键）打开了image的预览模式时，与image viewer的左右切换冲突了，想要定制化的去控制

目前有一个close事件，但是缺少一个show的事件

### Additional comments

```vue
<script setup>
defineProps({ /* ... */ })
</script>

<template>
  <el-image @close="close" @show="show"></el-image>
</template>
```

**Comment by btea** [comment]
resolved by #8920
